common/lib/xmodule/xmodule/js/src/video/02_html5_video.js

```diff
@@ -44,8 +44,11 @@ function(_) {
          *                            // video format of the source. Supported
          *                            // video formats are: 'mp4', 'webm', and
          *                            // 'ogg'.
+         *        poster:             Video poster URL
```

Comment:
`poster` is HTML 5 term and according to specs it must be an url thats why it is just named as `poster`